cmd/request_cert.go

```diff
@@ -82,6 +83,9 @@ func newCertAction(c *cli.Context) {
 	}
 
 	formattedName := strings.Replace(name, " ", "_", -1)
+	if c.IsSet("out-name") {
+		formattedName = strings.Replace(c.String("out-name"), " ", "_", -1)
```

Comment:
I think the string replace is potentially confusing and we should omit it.  Just do `formattedName = out-name`.  If the user specifies an out-name, we should use it directly.